### ISSUE TITLE
Equipment system

### ISSUE THREAD

**Issue by Vosvek** [Vosvek/ProjectRogue#3]
Implement equipment system. Currently, only `Actor` will use this. This might also result in `Equippable` stubs. Remember, Mechanisms should interact only within themselves.

Expect the Actor to have its own way of adding/equipping items to and from the inventory.

**Comment by Vosvek** [comment]
Progressing. Currently waiting on the Stat mechanism.

**Comment by Vosvek** [comment]
Currently on hold.

The remaining features are damage, conditions, and resistance (all of which are waiting for turn actions and the turn manager for testing), and equipment sprites (waiting for art and a possible refactor).
